src/scripts/main.js

```diff
@@ -1,5 +1,25 @@
 'use strict';
 
+const field = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+const fieldWidth = field.clientWidth;
+const fieldHeight = field.clientHeight;
+const spiderWidth = spider.clientWidth;
+const spiderHeight = spider.clientHeight;
+
 document.addEventListener('click', (e) => {
-  // write code here
+  const x = e.clientX;
+  const y = e.clientY;
```

Comment:
Ensure that the click coordinates (x, y) are correctly calculated relative to the viewport. If the field element is not positioned at the top-left corner of the viewport, the calculations for newX and newY might need adjustments.